client.py

```diff
@@ -0,0 +1,34 @@
+import socket
+
+ADDR = ('127.0.0.1', 8011) #port 0 may force os to find an open port
```

Comment:
The better solution is to use SO_REUSEADDR to allow the port to be immediately re-used after quitting the server script.